iothub/device/src/Common/Exceptions/IotHubSuspendedException.cs

```diff
@@ -8,7 +8,8 @@
 namespace Microsoft.Azure.Devices.Client.Exceptions
 {
     /// <summary>
-    /// The exception that is thrown when the IoT Hub has been suspended.
+    /// This exception is thrown when the IoT Hub has been suspended. This is likely due to exceeding your
```

Comment:
Casing should be "IoT hub" with a lowercase 'h' per style guidelines.

Reply:
FWIW, I prefer not using 2nd person in errors. Feels too personal. I think it sounds less accusatory to phrase it like "This is likely due to exceeding Azure spending limits. To resolve the error, check the Azure bill and ensure there are enough credits."

Reply:
Any suggestions on wording here? "This error is likely due to exceeding Azure spending limits. This error will be resolved when where is enough credits in the Azure account." ?

Reply:
Cool. I'll update.